naturewatch_camera_server/data.py

```diff
@@ -83,6 +69,25 @@ def delete_video(filename):
             return Response('{"ERROR": "' + filename + '"}', status=500, mimetype='application/json')
 
 
+def get_all_files(current_app, src_path):
+    # just for now... we should take an array of file names
+    src_list = construct_directory_list(current_app, src_path)
```

Comment:
@mikevanis here you could provide a list of filenames into `src_list` instead.

Reply:
Great, we could probably generate a list with the ContentSelector that's currently used to "Delete Selected".